build.gradle

```diff
@@ -79,3 +79,36 @@ processResources {
 jar {
     from configurations.shade.collect { it.isDirectory() ? it : zipTree(it) }
 }
+
+// utility tasks
+task deploy {
+	group = 'build'
+	description = 'Generates a jar and a source jar for release'
+	
+	dependsOn jar
```

Comment:
This needs to depend on `assemble` instead, otherwise the `retromapReplacedMain` task will not run.